mkosi/__init__.py

```diff
@@ -4043,6 +4030,46 @@ def check_workspace_directory(config: Config) -> None:
                 hint="Use WorkspaceDirectory= to configure a different workspace directory")
 
 
+def run_clean_scripts(config: Config) -> None:
+    if not config.clean_scripts:
+        return
+
+    for script in config.clean_scripts:
+        if not os.access(script, os.X_OK):
+            die(f"{script} is not executable")
+
+    env = dict(
+        DISTRIBUTION=str(config.distribution),
+        RELEASE=config.release,
+        ARCHITECTURE=str(config.architecture),
+        SRCDIR="/work/src",
+        OUTPUTDIR="/work/out",
+        MKOSI_UID=str(INVOKING_USER.uid),
+        MKOSI_GID=str(INVOKING_USER.gid),
```

Comment:
`MKOSI_CONFIG` is missing here
```suggestion
        MKOSI_GID=str(INVOKING_USER.gid),
        MKOSI_CONFIG="/work/config.json",
```